Week-2/Exercise-2.1/README.md

```diff
@@ -0,0 +1,14 @@
+Exercise2.1:(2-3hours)
+Create a new webpage on a subject that you're interested in. The page should **NOT** use anyframework like Bootstrap and it should include the following elements:
+- The `.html` page and a linked `.css` file applies a consistent color scheme to links and text onyour page.
+- - A page header with a title and description for the site.
+- - An articles section with three articles, each including a title, summary and a link.
+- - A page footer containing info about you or the site.
+- - Proper use of the semantic HTML tags we discussed.
+- - Use this website [Lorem Ipsum](http://www.lipsum.com/) to generate random blocks ofcontent to fill your website.
+- - **Commit often (with good meaningful commit messages), and Push to Github regularly.**
+- -  Add a form to your site. This form can contain as many fields as you wish to have but makesure it follows proper use of the semantic HTML tags we discussed.  Add validation to thesefields without using any framework or JavaScript.
+- - On a larger device we should see a 2 column layout. On a smaller device the right sidecontent should stack below the image.  Add at least three more articles and then convert themto a grid, so that you show 3 articles per row in a box layout.
```

Comment:
This point is not implemented. On smaller devices, it should show everything in a single column.

Reply:
Changes are made.